libafl/src/executors/forkserver.rs

```diff
@@ -573,10 +633,21 @@ impl Forkserver {
     }
 }
 
-/// This [`Executor`] can run binaries compiled for AFL/AFL++ that make use of a forkserver.
+/// An executor that runs LibAFL-instrumented binaries via a forkserver protocol
+///
+/// This executor communicates with instrumented targets using the forkserver mechanism
+/// for efficient process creation. It supports shared memory-based test case passing,
+/// customizable timeouts, and can handle persistent mode execution.
+///
+/// For persistent mode details, see: 
+/// <https://github.com/AFLplusplus/AFLplusplus/blob/stable/instrumentation/README.persistent_mode.md>
 ///
-/// Shared memory feature is also available, but you have to set things up in your code.
-/// Please refer to AFL++'s docs. <https://github.com/AFLplusplus/AFLplusplus/blob/stable/instrumentation/README.persistent_mode.md>
+/// # Type Parameters
```

Comment:
I think if we start documenting type parameters, they will eventually be refactored and forgotten.
Instead we should document common letters for things in a common place, and drop it from this function.